install/0000_30_machine-api-operator_01_images.configmap.yaml

```diff
@@ -15,7 +15,7 @@ data:
       "clusterAPIControllerAWS": "registry.svc.ci.openshift.org/openshift:aws-machine-controllers",
       "clusterAPIControllerAlibaba": "registry.svc.ci.openshift.org/openshift:alibaba-machine-controllers",
       "clusterAPIControllerOpenStack": "registry.svc.ci.openshift.org/openshift:openstack-machine-controllers",
-      "clusterAPIControllerMAPO": "quay.io/shiftstack/machine-api-provider-openstack:latest",
+      "clusterAPIControllerMAPO": "quay.io/openshift/origin-machine-api-provider-openstack",
```

Comment:
Looking at the org, looks like this image hasn't actually been created yet, is the image mirroring in flight?

Reply:
yeah: https://github.com/openshift/release/pull/24933

Reply:
Is it expected the new image isn't in registry.svc.ci.openshift.org like the other ones on that list?

Reply:
according to mike, the correct repo is actually quay for all of them: https://github.com/openshift/machine-api-operator/pull/881